platform-controller/src/main/java/org/hobbit/controller/config/HobbitConfig.java

```diff
@@ -0,0 +1,43 @@
+package org.hobbit.controller.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * Created by Tim Ermilov on 15.05.17.
+ */
+public class HobbitConfig {
+    public class TimeoutConfig {
+        public long benchmarkTimeout;
+        public long challengeTimeout;
+
+        public TimeoutConfig(long bt, long ct) {
+            benchmarkTimeout = bt;
+            challengeTimeout = ct;
+        }
+    }
```

Comment:
It needs to be documented that a user always has to define both timeouts. What happens if only one of them is defined? Will the parsing in line `loadConfig` fail?